core/src/main/scala/kafka/log/ProducerStateManager.scala

```diff
@@ -354,6 +357,7 @@ object ProducerStateManager {
   private val VersionOffset = 0
   private val CrcOffset = VersionOffset + 2
   private val ProducerEntriesOffset = CrcOffset + 4
+  protected lazy val logger = Logger(LoggerFactory.getLogger(this.getClass))
```

Comment:
Instead of this can you extend the object from Logging class. That would be much cleaner

**object ProducerStateManager extends Logging**

Reply:
I will try in this way. The way I am using doesn't seem working.